roles/kubernetes/master/templates/manifests/kube-apiserver.manifest.j2

```diff
@@ -41,7 +41,7 @@ spec:
     - --advertise-address={{ ip | default(ansible_default_ipv4.address) }}
     - --etcd-servers={{ etcd_access_addresses }}
 {%   if etcd_events_cluster_enabled %}
-    - --etcd-servers-overrides=/events#{{ etcd_events_access_addresses }}
+    - --etcd-servers-overrides=/events#{{ etcd_events_access_addresses_semicolon }}
```

Comment:
errrr...... the list is seperated by colon, where the groups are seperated by semi colon ?

Reply:
?